tox.ini

```diff
@@ -89,3 +89,13 @@ commands =
         --confcutdir=tests/integration \
         {posargs:tests/integration}
 passenv = KRB5CCNAME REQUESTS_CA_BUNDLE KRB5_CLIENT_KTNAME CACHITO_TEST_CERT CACHITO_TEST_KEY JOB_NAME
+
+[pytest]
+addopts = --tb=native --verbose --capture=no
+testpaths = tests
+log_cli = True
+log_level = NOTSET
```

Comment:
IIUC the purpose of this is, in fact, to _enable_ logging? NOTSET kinda implies the opposite to me. Maybe set it to DEBUG?

Reply:
yes only purpose of this section is enable logging I can change it to DEBUG